### PR TITLE
Allow overriding layer srid

### LINKED ISSUE #528: MapConfig's mapnik layer SRID not taken into account
It looks like the layer.options.srid is not being added to extra_ds_opts passed to grainstore, when getting a new renderer. This in turn prevents the MapConfig specification for being supported in that reguard.
See https://github.com/CartoDB/Windshaft/blob/master/doc/MapConfig-1.5.0.md#21-mapnik-layers-options

### PR DESCRIPTION
**Pull request by dgaubert**
Fixes #528 and supersedes #529 

Roughly, https://github.com/CartoDB/Windshaft/commit/e0f6b49b3e8aa2c8719ae72f6a8a7f4e40e0e628 is what @rochoa meant.

cc/ @strk

### CODE REVIEW

**Comment by rochoa** [comment]
I was _sketching_ similar changes, I just merged with yours. I think we have to add a test and we are good to :shipit:, what do you think?

File: lib/windshaft/models/mapconfig.js [hunk 1]
```diff
@@ -170,16 +170,25 @@ function getLayerIndexByType(rawMapConfig, type, mapConfigLayerIdx) {
     return mapConfigToTypeLayers[mapConfigLayerIdx];
 }
 
+MapConfig.prototype.getLayerOption = function(layerIndex, optionName, defaultValue) {
```
Comment (dgaubert) [inline lib/windshaft/models/mapconfig.js]: ❤️

**Comment by dgaubert** [comment]
What do you think about upgrading `grainstore` to 1.6.0?

**Comment by dgaubert** [comment]
I've just added some test for that

**Pull request merged**